src/schema/generated-schema-types.ts

```diff
@@ -1480,6 +1480,13 @@ export type ExternalAccountTestResponse = {
   success: Scalars['Boolean'];
 };
 
+export enum ExternalSendersType {
+  All = 'ALL',
+  Allnotinab = 'ALLNOTINAB',
```

Comment:
This property name is not readable at all, please make it more readable by following standard naming convention.

Reply:
This is an auto-generated file. These values are the same as those defined in the schema. We can add comments in the schema for better understanding.

Reply:
Got it. I don't think adding comment here for each property is good idea and will not improve anything in zm-x-web code where we have used it multiple places, you can leave it as it is.